django/cantusdb_project/main_app/models/base_chant.py

```diff
@@ -59,6 +60,25 @@ class Meta:
         "Office", on_delete=models.PROTECT, null=True, blank=True
     )
     position = models.CharField(max_length=63, null=True, blank=True)
+
+    BENEDICAMUS_DOMINO = "BD"
+    PROCESSIONAL = "P"
+    HISTORIAE = "H"
+    DRAMATIC_ELEMENT = "D"
```

Comment:
Why do we need to define these variables outside of `LITURGICAL_FUNCTION_CHOICES` if their full names are still represented in the list?

Reply:
I followed the model [https://docs.djangoproject.com/en/5.0/ref/models/fields/#choices](https://docs.djangoproject.com/en/5.0/ref/models/fields/#choices). 

The string actually stored in the database is the first element of each tuple, and the second element is the "public-facing name". My understanding of defining them separately is so that you can do something like:

```python
from main_app.models import Chant
chants_processional = Chant.objects.filter(liturgical_function = Chant.PROCESSIONAL)
```

rather than having to remember that PROCESSIONAL is stored as "P" in the database. But you certainly don't *need* to do that

Reply:
Oh okay makes sense! Let's keep it this way then.